pkg/solidfire/types.go

```diff
@@ -12,9 +12,10 @@ var (
 	EndpointFlag          = "endpoint"
 	InsecureSSLFlag       = "insecure"
 	HTTPClientTimeoutFlag = "timeout"
-	ListenPortFlagEnv     = "SOLIDFIRE_PORT"
+	ConfigFileFlag        = "config.yaml"
```

Comment:
This is the flag _name_ not the flag's default value.

You probably meant this to be as such:

```suggestion
	ConfigFileFlag        = "config"
```